pkg/buildClient/comments.go

```diff
@@ -59,6 +63,10 @@ func prComments(buildSystem string, result []*buildsecurity.Result, avdUrlMap Re
 	default:
 		return nil
 	}
+	err := c.RemovePreviousAquaComments(aquaMsg)
+	if err != nil {
+		log.Logger.Infof("failed removing old comments with error: %s", err)
```

Comment:
Let's add it also to failed PR comment instead of returning an error please

Reply:
You mean changing to return fmt.Errorf("failed ***: %w", err).
I thought we shouldnt failt the whole writing because of this?

Reply:
Added to all error, just to print, so we get the next comments